kernel/kernel-api/src/main/java/io/delta/kernel/metrics/SnapshotMetricsResult.java

```diff
@@ -15,9 +15,11 @@
  */
 package io.delta.kernel.metrics;
 
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import java.util.Optional;
 
 /** Stores the metrics results for a {@link SnapshotReport} */
+@JsonPropertyOrder({"timestampToVersionResolutionDurationNs", "loadInitialDeltaActionsDurationNs"})
```

Comment:
Could you tell me about this? (As a reply to this comment; don't need you to add anything to the code).

Reply:
I found in some test runs the order the fields were serialized in was not deterministic so added this.